include/AAInterface.hpp

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "openauto/Service/IAndroidAutoInterface.hpp"
+#include "app/action.hpp"
+
+
+
+class AAInterface : public QObject, public openauto::service::IAndroidAutoInterface {
```

Comment:
maybe rename this to AndroidAutoHandler or something else? (just a bit confusing since you're already inheriting from an "interface")

Reply:
Yea, that's a good idea - I can see this being confusing.